src/python/cctrusted_vm/cvm.py

```diff
@@ -391,6 +393,64 @@ def get_cc_report(self, nonce: bytearray, data: bytearray, extraArgs) -> CcRepor
         self.process_cc_report(input_data)
         report_bytes = self.tdreport.data
 
+        if self.version is TDX_VERSION_1_0:
+            quote_req = TdxQuoteReq10()
+        elif self.version is TDX_VERSION_1_5:
+            quote_req = TdxQuoteReq15()
+
+        tdvmcall_flag = False
+        # Use vsock to get TD Quote by default
+        with open(TdxVM.CFG_FILE_PATH, 'rb') as port_file:
+            port_info = port_file.read().rstrip().decode("utf-8")
+            if "port=" not in port_info:
```

Comment:
Hmm, it will be false when I configure it to "port =xx" or "port<tab>=xx", right?

Reply:
Yes. Changed by trimming all spaces and do the comparsion.